src/rai_toolbox/mushin/workflows.py

```diff
@@ -92,31 +116,47 @@ def __init__(self, eval_task_cfg=None) -> None:
         self.jobs = []
         self.working_dir = Path.cwd()  # TODO: I don't think we should assume this
```

Comment:
I think this should be `None` and require that it either be set by the user, `load_from_dir`, or after `run`

Reply:
Agreed, I changed it to default to `None`